docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -79,6 +79,27 @@ Auth Service's gRPC endpoint.
 
 The Helm chart only supports the `file` format.
 
+```code
+$ tctl auth sign --user=access-plugin --out=auth
+```
+
+Then create a Kubernetes secret:
+
+```code
+$ kubectl create secret generic teleport-mattermost-identity --from-file=auth_id=auth.pem
+```
+
+Teleport's Access Request plugins listen for new and updated Access Requests by
+connecting to the Teleport Auth Service's gRPC endpoint over TLS.
+
+The identity file, `auth.pem`, includes both TLS and SSH credentials. Your
+Access Request plugin uses the SSH credentials to connect to the Proxy Service,
+which establishes a reverse tunnel connection to the Auth Service. The plugin
+uses this reverse tunnel, along with your TLS credentials, to connect to the
+Auth Service's gRPC endpoint.
+
+The Helm chart only supports the `file` format.
```

Comment:
```suggestion
```
Since the `tctl auth sign` command uses the default format, rather than an explicit `--format=file` flag, I think we can delete this.